packages/kit/src/views/Setting/pages/Tab/CustomElement.tsx

```
@@ -299,23 +299,13 @@ export function HardwareTransportTypeListItem(props: ICustomElementProps) {
-            label: 'Bluetooth',
-            value: EHardwareTransportType.DesktopWebBle,
-          },
-        ];
-      }
       return [
         {
-          label: 'Bridge',
-          value: EHardwareTransportType.Bridge,
+          label: 'WebUSB',
+          value: EHardwareTransportType.WEBUSB,
+          iconProps: { name: 'UsbOutline' as const }
```

ESLint

[error] Insert `,`

(prettier/prettier)